Source/UI/Screens/Password/PasswordViewController.swift

```diff
@@ -137,7 +137,7 @@ class PasswordViewController: IdentityUIViewController {
     }
 
     @IBAction func didClickContinue(_: Any) {
-        guard let password = self.password.text, password.count >= 8 else {
+        guard let password = self.password.text, ((self.viewModel.loginFlowVariant == .signin && password.count >= 1) || password.count >= 8) else {
```

Comment:
nitpicking: it's a bit difficult to understand what you are trying to achieve here, too complex logic for a guard clause

Reply:
Can't argue with that :)